charts/incubator/alfresco-dbp/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: alfresco-dbp
-version: 1.5.0
+version: 1.6.0
```

Comment:
I believe this should stay at 1.5.0, see https://w3.alfresco.com/confluence/display/DBP/DBP+Version+Management+Guide

Reply:
Apologies, I forgot we had appVersion as well in this chart already.  Disregard.